tests/config/konfig-local.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
Duplicate file? Couln't it be copied from https://github.com/geokrety/geokrety-website/blob/master/configs/konfig-local.tmpl.php as you did for the sql files ;)

Reply:
yep good point, done